src/ui/citation-popup/view.js

```diff
@@ -21,8 +21,19 @@ const output = document.getElementById("output");
 function main () {
 
     /* State */
+    const refs = null;
+    try { refs = JSON.parse(inputRefsView.textContent); } catch {}
```

Comment:
refs should be defined with `let refs` since you assigned a variable to it afterwards.

Also in general avoid empty catch statements as it makes it very to debug when there's problem. At least display a console statement.